src/components/CommentsList/CommentsList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+type Props = {
+  comment: Comment[];
+};
+
+export const CommentsList: React.FC<Props> = ({ comment }) => (
+  <ul className="CommentsList" data-cy="comments-list">
+    {comment.map((item) => (
+      <li key={item.id}>
+        <CommentInfo
+          name={item.name}
+          email={item.email}
+          body={item.body}
+        />
+      </li>
+    ))}
```

Comment:
what happens if no one has commented yet?
it would be nice to handle this situation like you did with userInfo